tests/tests/listable.rs

```diff
@@ -9,6 +9,8 @@ impl Visit for VisitHello {
         assert_eq!("id", id.name());
         assert_eq!(Some(self.0), named_values.get(id).unwrap().as_u32());
     }
+
+    fn visit_value(&mut self, _: Value<'_>) {}
```

Comment:
if we don't expect this to ever be called in the test should it maybe be
```suggestion
    fn visit_value(&mut self, _: Value<'_>) {
        unreachable!("not called in this test");
    }
```
or something? not a big deal; i can't imagine why this would ever start failing, but it could be good?